src/util/qwlogging.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2024 WenHao Peng <pengwenhao@uniontech.com>.
+// SPDX-License-Identifier: Apache-2.0 OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+
+#include "qwlogging.h"
+
+#include <QDebug>
+#include <QLoggingCategory>
+
+QW_BEGIN_NAMESPACE
+
+Q_LOGGING_CATEGORY(QWlroots, "qwlroots");
+
+void QWLog::init()
+{
+#ifdef QT_DEBUG
+    wlr_log_init(WLR_DEBUG, log_callback);
```

Comment:
Always enable debug here. We can control default log level via Q_LOGGING_CATEGORY.